src/bin/pg_autoctl/pgsetup.c

```diff
@@ -160,12 +155,7 @@ pg_setup_init(PostgresSetup *pgSetup,
 	}
 	else
 	{
-		char *pguser = getenv("PGUSER");
-
-		if (pguser)
-		{
-			strlcpy(pgSetup->username, pguser, NAMEDATALEN);
-		}
+		(void) get_env_variable("PGUSER", pgSetup->username, NAMEDATALEN);
```

Comment:
Why is there no check for errors here? If that's on purpose, let's add a comment.